test/javascripts/acceptance/unknown-test.js.es6

```diff
@@ -1,8 +1,49 @@
 import { acceptance } from "helpers/qunit-helpers";
+import pretender from "helpers/create-pretender";
 acceptance("Unknown");
 
 QUnit.test("Unknown URL", async assert => {
+  pretender.get("/permalink-check.json", () => {
+    return [404, {}, null];
+  });
   assert.expect(1);
   await visit("/url-that-doesn't-exist");
   assert.ok(exists(".page-not-found"), "The not found content is present");
 });
+
+QUnit.test("Permalink URL to a Topic", async assert => {
+  pretender.get("/permalink-check.json", () => {
+    return [
+      200,
+      { "Content-Type": "application/json" },
+      {
+        found: true,
+        internal: true,
+        target_url: "/t/internationalization-localization/280"
+      }
+    ];
+  });
+
+  await visit("/viewtopic.php?f=8&t=280");
+  assert.ok(exists(".topic-post"));
+});
+
+QUnit.test("Permalink URL to 'outside' the site", async assert => {
+  // TODO: This is a fake test, the server will never actually return this
+  // We can't capture window.location= without some dirty mocks, though.
+  pretender.get("/permalink-check.json", () => {
+    return [
+      200,
+      { "Content-Type": "application/json" },
+      {
+        found: true,
+        internal: true,
+        target_url: "/faq"
+      }
+    ];
+  });
+
+  await visit("/not-the-url-for-faq");
+  assert.ok($("body.static-faq").length, "routed to the faq page");
```

Comment:
We try to not use jquery as much as possible

```suggestion
  assert.ok(exists("body.static-faq"), "routed to the faq page");
```

Reply:
For some reason this started failing...

Reply:
Is there something special about $ in tests that makes it work for `body` ?

Reply:
Yes I think I know what's going on.

find is probably scoped to #ember-testing-container when $ is scoped to document.

So I guess this is a special case. In this case you could do : 

```
// body is outside of #ember-testing-container and needs to be targeted
// through document instead of find
assert.ok(document.querySelector("body.static-faq"), "routed to the faq page");
```

I think that should work.

Reply:
Yep, this worked!